test/sql/cluster.sql

```diff
@@ -0,0 +1,24 @@
+CREATE TABLE cluster_test(time timestamptz, temp float);
+
+SELECT create_hypertable('cluster_test', 'time', chunk_time_interval => interval '1 day');
+
+-- Show default indexes
+SELECT * FROM test.show_indexes('cluster_test');
+
+-- Create two chunks
+INSERT INTO cluster_test VALUES ('2017-01-20T09:00:01', 23.4),
+       ('2017-01-21T09:00:01', 21.3);
+
+-- Run cluster
+CLUSTER VERBOSE cluster_test USING cluster_test_time_idx;
+
+-- Show clustered indexes
+SELECT indexrelid::regclass, indisclustered
+FROM pg_index
+WHERE indisclustered = true;
+
+-- Recluster just our table
+CLUSTER VERBOSE cluster_test;
+
+-- Recluster all tables (although will only be our table)
+CLUSTER VERBOSE;
```

Comment:
I would like to see a test changing the clustered index. Also a test that clusters chunks and not hypertables (especially when not specifying the clustered index). 

Related to that: I think we need to check creating a new chunk after setting the clustered index. Would the following work:

1) create a hypertable
2) create two chunks
3) CLUSTER VERBOSE cluster_test USING cluster_test_time_idx;
4) create a third chunk
5) CLUSTER VERBOSE cluster_test

would the third chunk know which index is the clustered one?